### PR TITLE
Sample .Net WebAPI with Amazon DynamoDB & Amazon DynamoDB Accelerator

### PR DESCRIPTION
**Pull request by srcsakthivel**
**Description of changes:**

Sample .Net WebAPI with Amazon DynamoDB & Amazon DynamoDB Accelerator. API write DynamoDB and Read it through DynamoDB Accelerator.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Storage/DynamoDbDAX/dydb.api/appsettings.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "DAXEndpoint": "daxs://dax-workshop.nkrbmv.dax-clusters.us-east-1.amazonaws.com"
```
Comment (isaacrlevin) [inline Storage/DynamoDbDAX/dydb.api/appsettings.json:9]: Remove your endpoint here.

**Comment by isaacrlevin** [comment]
Hey there @srcsakthivel thanks for submitting a PR! Can you please add a more detailed README?

**Comment by srcsakthivel** [comment]
> Hey there @srcsakthivel thanks for submitting a PR! Can you please add a more detailed README?

Updated and removed the Endpoint.

File: Storage/DynamoDbDAX/dydb.api/appsettings.json (hunk 1 above)
Comment (srcsakthivel) [reply Storage/DynamoDbDAX/dydb.api/appsettings.json]: Updated

**Comment by srcsakthivel** [comment]
@isaacrlevin - Let me know if you are ok with updates.

**Comment by isaacrlevin** [comment]
This looks really good!
Can you also add screens to get the connection string as well as more detail about the info at the bottom?

**Comment by srcsakthivel** [comment]
@isaacrlevin - added the screen and details. Hope that helps.

**Comment by isaacrlevin** [comment]
Thanks for the contribution!

**Pull request merged**